feat: Implement game sessions feature

Cybermaxi7 opened the pull request:
- Add GameSession entity with relations to User
- Implement CRUD operations for game sessions
- Add leaderboard and recent games functionality
- Add migration for game_sessions table
- Update User entity with game sessions relation
CLOSES #46

Xaxxoo reviewed (approved):
LGTM

godwinameh commented:
Thanks 🙏

On Thu, Mar 20, 2025, 10:40 PM Xaxxoo ***@***.***> wrote:

> Merged #71 <https://github.com/songifi/lyricsflip_server/pull/71> into
> main.
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/songifi/lyricsflip_server/pull/71#event-16918424849>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/BBUZYBZGM5YRKYBQDB5NX4L2VMYVFAVCNFSM6AAAAABZONGNN6VHI2DSMVQWIX3LMV45UABCJFZXG5LFIV3GK3TUJZXXI2LGNFRWC5DJN5XDWMJWHEYTQNBSGQ4DIOI>
> .
> You are receiving this because you authored the thread.Message ID:
> ***@***.***>
>

codesage7 committed:
feat: Implement game sessions feature

- Add GameSession entity with relations to User
- Implement CRUD operations for game sessions
- Add leaderboard and recent games functionality
- Add migration for game_sessions table
- Update User entity with game sessions relation

Xaxxoo reviewed (approved):
LGTM
